src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -329,6 +352,89 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
     }
   }
 
+  /**
+   *  media Stream metrics
+   */
+
+  readonly observableVideoMetricSpec: {
+    [id: string]: {
+      source: string;
+      media?: MediaType;
+      dir?: Direction;
+    };
+  } = {
+    videoUpstreamBitrate: {
+      source: 'bytesSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamPacketsSent: {
+      source: 'packetsSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamFramesEncodedPerSecond: {
```

Comment:
What is the equivalent in Goog?

Reply:
The equivalent in Goog is framesEncoded in inbound-rtp and outbound-rtp report.